teknologiradar.json

```
@@ -163,6 +163,21 @@
         }
       ]
     },
+    {
+      "key": "Spring-framework",
+      "title": "Spring Framework",
+      "ring": "avstå",
+      "quadrant": "programmering",
+      "description": "Rammeverk for utvikling av backendtjenester. Spring er et stort rammeverk med mye magi som kan være komplisert å konfigurere riktig. Vurder derfor Micronaut før man tar i bruk Spring på nye tjenester. <br> Brukes av SUP og Freg",
```

```suggestion
      "description": "Rammeverk for utvikling av backendtjenester. Spring er et stort rammeverk med mye magi som kan være komplisert å konfigurere riktig. Vurder derfor Micronaut før man tar i bruk Spring på nye tjenester. Brukes av SUP og Freg",
```

HTML rendres ikke i Backstage sin Tech Radar. Det er noe HTML fra forrige radar, men det må også ryddes opp.